setup.py

```diff
@@ -11,6 +11,7 @@
     'requests>=1.0',
     'python-dateutil>=2.1',
     'six>=1.2.0',
+    'akamai-edgeauth==0.3.2',
```

Comment:
If it's not too much to ask, I suggest to add this dep to "akamai" extra. Not sure if signature generation logic or file structure will be affected by this, though.